sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/ConfigurationSettingsSnapshot.cs

```diff
@@ -14,7 +14,7 @@ public partial class ConfigurationSettingsSnapshot
         /// <summary> Initializes a new instance of Snapshot. </summary>
         /// <param name="filters"> A list of filters used to filter the key-values included in the snapshot. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="filters"/> is null. </exception>
-        public ConfigurationSettingsSnapshot(IEnumerable<ConfigurationSettingFilter> filters)
+        public ConfigurationSettingsSnapshot(IEnumerable<SnapshotSettingFilter> filters)
```

Comment:
>SnapshotSettingFilter [](http://example.com/codeflow?start=57&length=21)

Just curious, what is the motivation of this name change? ConfigurationSettingFilter seemed more aligned in terms of naming to the class name than snapshot setting filter.

Reply:
The reasoning was that the setting filter is only used by the snapshot feature, and it seemed unclear that that was the case without having snapshot in the name. The other alternative we considered was `ConfigurationSettingsSnapshotSettingFilter` but decided instead that all of the supplementary types would just be `Snapshot...` and then only the snapshot object would include configuration in the name (to avoid a single name type)